test/lib/build/TaskRunner.js

```diff
@@ -851,6 +851,67 @@ test("Custom task with specVersion 3.0", async (t) => {
 	}, "Task got called with one argument");
 });
 
+test("Custom task with specVersion 3.0 and no requiredDependenciesCallback", async (t) => {
+	const {sinon, graph, taskUtil, taskRepository, TaskRunner} = t.context;
+	const taskStub = sinon.stub();
+	const specVersionGteStub = sinon.stub().returns(true);
+	const mockSpecVersion = {
```

Comment:
do we have a test for specVersion < 3.0?

Reply:
Yes. I was surprised I missed to add this test, all the other combinations seem to be covered.

specVersion < 3.0 without callback: https://github.com/SAP/ui5-project/pull/536/files/c34a6628f3c75f107ce65fd41f05aba663a998e7#diff-27d98157bc20195d53765868ae91b0d166a863e5c582861f2bfa316c7a0bcbd1R631

Reply:
Ack